test/cmocka/src/audio/mux/mux_copy.c

```diff
@@ -100,31 +100,52 @@ static int setup_group(void **state)
 	return 0;
 }
 
+static size_t get_stream_map_size(struct test_data *td)
+{
+	size_t size = 0;
+	int i, j;
+
+	for (i = 0; i < MUX_MAX_STREAMS; ++i) {
```

Comment:
Looks like common code now, care to deduplicate.

Reply:
Moved to util.h